sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ChooseSourceCaseSteps.java

```diff
@@ -97,11 +99,36 @@ public ChooseSourceCaseSteps(
               .assertThat(apiState.getCreatedCase().getCaseClassification())
               .isEqualToIgnoringCase(caseClassification);
           softly
-          // this substring method will return the first 6 characters from the UUID.
-          // those characters are used in UI as the Case ID.
+              // this substring method will return the first 6 characters from the UUID.
+              // those characters are used in UI as the Case ID.
               .assertThat(apiState.getCreatedCase().getUuid().substring(0, 6))
               .isEqualToIgnoringCase(caseId);
           softly.assertAll();
         });
+
+    When(
+        "I click on the CHANGE CASE button",
+        () ->
+            webDriverHelpers.clickWhileOtherButtonIsDisplayed(
+                CHANGE_CASE_BUTTON, POPUP_YES_BUTTON));
+
+    When(
+        "I click on the Remove Case CTA",
+        () ->
+            webDriverHelpers.clickWhileOtherButtonIsDisplayed(
+                REMOVE_CASE_CTA_LINK, POPUP_YES_BUTTON));
+
+    Then(
+        "I check the CHOOSE SOURCE CASE BUTTON is displayed",
+        () ->
+            assertWithMessage("The expected element was not displayed")
+                .that(webDriverHelpers.isElementVisibleWithTimeout(CHOOSE_SOURCE_CASE_BUTTON, 1))
+                .isTrue());
+
+    When(
+        "I click yes on the CONFIRM REMOVAL popup",
```

Comment:
also, add the location of this removal popup

Reply:
reworked